common-service-core/src/main/java/com/ca/mfaas/utils/UrlUtils.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package com.ca.mfaas.utils;
+
+import lombok.experimental.UtilityClass;
+import org.apache.commons.lang.RandomStringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.*;
+import java.util.function.Supplier;
+
+
+@UtilityClass
+public class UrlUtils {
+    private static final Logger logger = LoggerFactory.getLogger(UrlUtils.class);
+
+    private static Supplier<String> messageSupplier = new Supplier() {
```

Comment:
`private static Supplier<String> messageSupplier = new Supplier<String>() {`
The expression of type new Supplier(){} needs unchecked conversion to conform to Supplier<String>